dz_s4/zadacha_7_from_seminar.py

```diff
@@ -0,0 +1,15 @@
+# Функция получает на вход словарь с названием компании в качестве ключа
+# и списком с доходами и расходами (3-10 чисел) в качестве значения.
+# ✔ Вычислите итоговую прибыль или убыток каждой компании. Если все компании
+# прибыльные, верните истину, а если хотя бы одна убыточная — ложь.
+
+def func(some_dict):
+    for name in some_dict:
+        some_dict[name] = sum(some_dict[name])
+    print(all([x > 0 for x in some_dict.values()]))
```

Comment:
Функция all принимает простой генератор.
```suggestion
    print(all(x > 0 for x in some_dict.values()))
```